src/com/epam/task/module4/theSimplestClassesAndObjects/task1/test1/Test1.java

```diff
@@ -0,0 +1,22 @@
+package com.epam.task.module4.theSimplestClassesAndObjects.task1.test1;
+
+public class Test1 {
+    private Integer first = 2;
+    private Integer second = 4;
```

Comment:
Suggestion: do not initialize the variable immediately after the announcement. Maybe the author, who would like to create an object of this class, wanted different default values and could specify them during the creation of the object in the constructor.

Reply:
At the same time, for this task it is ok.